src/components/InputForm/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+import classes from './style.module.css';
+
+class InputForm extends Component {
```

Comment:
Did you test whether this works? I actually don't know if Gatsby supports ES7 classes. If so, then I think we should use them throughout everywhere.

Reply:
test what exactly? The import of classes? It is working on my side. I actually didn't know that that was ES7. But my styling is working.